docs/roman/datamodels/models.rst

```diff
@@ -8,58 +8,84 @@ the underlying file format.  The same data model may be used for data
 created from scratch in memory, or loaded from ASDF files or some future file
 format.
 
+The detailed datamodel structure and specifics are contained in the
+documentation included with the roman_datamodels package found here (TBD).
```

Comment:
`models` is an overloaded term. I would change the title of the section to be `About datamodels`.